ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx

```diff
@@ -1,20 +1,31 @@
 import * as React from "react"
-import { t } from "../../../helpers/translator"
 import { Listing } from "@bloom-housing/backend-core/types"
+import Markdown from "markdown-to-jsx"
+import { t } from "../../../helpers/translator"
+import { ExpandableContent } from "../../../actions/ExpandableContent"
 
 interface WhatToExpectProps {
   listing: Listing
 }
 
 const WhatToExpect = (props: WhatToExpectProps) => {
   const listing = props.listing
-
+  if (!listing.whatToExpect || listing.whatToExpect === "") return null
   return (
     <section className="aside-block">
       <h4 className="text-caps-underline">{t("whatToExpect.label")}</h4>
-      <p className="text-tiny text-gray-800">
-        {listing.whatToExpect ? listing.whatToExpect : t("whatToExpect.default")}
-      </p>
+      <div className="text-tiny text-gray-750">
```

Comment:
In the interests of future sync'ing of `ui-components` drift, do you think this change is written in a such a way we'd be able to  backport it to core?

Reply:
Updated 👍